translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/project/ProjectProgress.java

```diff
@@ -0,0 +1,129 @@
+package org.wycliffeassociates.translationrecorder.project;
+
+
+import android.content.Context;
+import com.door43.tools.reporting.Logger;
+import org.wycliffeassociates.translationrecorder.chunkplugin.Chapter;
+import org.wycliffeassociates.translationrecorder.chunkplugin.ChunkPlugin;
+import org.wycliffeassociates.translationrecorder.database.ProjectDatabaseHelper;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by mXaln on 7/31/2019.
+ */
+public class ProjectProgress {
+
+    Project project;
+    Context context;
+    ProjectDatabaseHelper db;
+
+    public ProjectProgress(Project project, Context context) {
+        this.project = project;
+        this.context = context;
+        this.db = new ProjectDatabaseHelper(context);
+    }
+
+    public int calculateProjectProgress() {
+        try {
+            ChunkPlugin chunks = project.getChunkPlugin(new ChunkPluginLoader(context));
```

Comment:
why not do this in the constructor, and move it to the top as a member variable since most every method here uses it?

Reply:
@sarabiaj That's a good idea. Updated!